automation/Dockerfile

```diff
@@ -21,16 +21,19 @@ RUN yum -y install epel-release centos-release-openstack-rocky
 RUN yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/nmstate/nmstate-test-env-el/repo/epel-7/nmstate-nmstate-test-env-el-epel-7.repo && \
     yum-config-manager --enable nmstate-nmstate-test-env-el
 
+# libibverbs is a dependency for openvswitch that is missing as a dependency in the RPM
 RUN yum -y upgrade && \
     yum -y install iproute python2-pytest python2-pytest-cov NetworkManager \
                    rpm-build git python2-devel python2-six \
                    python2-pyyaml python-jsonschema python-setuptools \
+                   NetworkManager-ovs openvswitch libibverbs \
                    python-gobject-base dnsmasq && \
     yum clean all && \
     install -o root -g root -d /etc/sysconfig/network-scripts && \
     echo -e "[logging]\nlevel=TRACE\ndomains=ALL\n" > /etc/NetworkManager/conf.d/97-docker-build.conf && \
     echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> /etc/NetworkManager/conf.d/97-docker-build.conf && \
-    sed -i 's/#RateLimitInterval=30s/RateLimitInterval=0/ ; s/#RateLimitBurst=1000/RateLimitBurst=0/' /etc/systemd/journald.conf
+    sed -i 's/#RateLimitInterval=30s/RateLimitInterval=0/ ; s/#RateLimitBurst=1000/RateLimitBurst=0/' /etc/systemd/journald.conf && \
+    systemctl enable openvswitch.service
```

Comment:
Why do we need for ovs but not for NM? NM enables it by default on installation?

Reply:
The systemd will start NetworkManager when got dbus access. The libnm is a wrapper of NM dbus interface.

Reply:
Not sure if Gris is also right but there is also a system preset to enable NetworkManager from what I saw, i.e. it will be enabled by default will not be.